gatsby-site/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "support-documents",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Support Documents",
+  "keywords": [
+    "gatsby"
+  ],
+  "scripts": {
+    "develop": "gatsby develop",
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean"
+  },
+  "dependencies": {
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
+    "babel-plugin-styled-components": "^1.12.0",
+    "gatsby": "^3.4.1",
```

Comment:
Should we bump to latest, v3.6.0?

Reply:
huh. yeah, sure. I'm surprised it didn't do that by default with a fresh init. _shrug_